TreeDB/collections/column_vector_graph_test.go

```
@@ -0,0 +1,460 @@
+	expectedFirstID := bytes.Clone(expected[0].DocumentID)
+	expectedFirstDistance := expected[0].Distance
+
+	const iterations = 100
+	workers := runtime.GOMAXPROCS(0)
+	if workers < 2 {
+		workers = 2
+	}
+	errs := make(chan error, workers)
+	var wg sync.WaitGroup
+	for worker := 0; worker < workers; worker++ {
+		worker := worker
```

golangci-lint (2.12.2)

[error] The copy of the 'for' variable "worker" can be deleted (Go 1.22+)

(copyloopvar)